src/x11.cpp

```diff
@@ -7,6 +7,29 @@
 #include <stdexcept>
 
 #include <X11/Xatom.h>
+#include <X11/Xresource.h>
+
+int getXresourceColor(const char* color, Display* display_)
```

Comment:
Nice idea, thanks!
Could you change the implementation a bit?
The function`getXresourceColor` can belong to the `x11` class, this will also eliminate the need to pass the `display_` variable.
I would suggest to create a private declaration in `x11.hpp` (next to the `redraw` function), something like that:
```
int getXresourceColor(const char* color) const;
```
and append the function body to the end of this file.

Or I can do it myself in this request, if you wish ;)